cmd/gapit/dump_fbo.go

```diff
@@ -78,42 +78,42 @@ func (verb *dumpFBOVerb) frameSource(ctx context.Context, client client.Client,
 	if err != nil {
 		return nil, log.Err(ctx, err, "Couldn't get filter")
 	}
+	filter.OnlyFramebufferObservations = true
 
-	return func(ch chan<- image.Image) error {
-		// Get the draw call and end-of-frame events.
-		events, err := getEvents(ctx, client, &path.Events{
-			Capture:                 capture,
-			LastInFrame:             true,
-			FramebufferObservations: true,
-			Filter:                  filter,
-		})
-		if err != nil {
-			return log.Err(ctx, err, "Couldn't get events")
+	treePath := capture.CommandTree(filter)
+
+	boxedTree, err := client.Get(ctx, treePath.Path(), nil)
+	if err != nil {
+		return nil, log.Err(ctx, err, "Failed to load the command tree")
+	}
+
+	tree := boxedTree.(*service.CommandTree)
+
+	var allFBOCommands []*path.Command
+	traverseCommandTree(ctx, client, tree.Root, func(n *service.CommandTreeNode, prefix string) error {
+		if n.Group != "" {
+			return nil
 		}
+		allFBOCommands = append(allFBOCommands, n.Commands.First())
+		return nil
+	}, "", true)
+
+	_ = allFBOCommands
+
+	return func(ch chan<- image.Image) error {
+
```

Comment:
I generally think whitespace increases the readability of code, especially since we now all use 1080p or larger screens. But reduced whitespace here to match the rest of the code.